apps/shared/avifutil.h

```diff
@@ -10,6 +10,59 @@
 extern "C" {
 #endif
 
+// NEXTARG(), MAIN(), INIT_ARGV(), FREE_ARGV() for UTF8-aware command line parsing.
+#if defined(_WIN32)
```

Comment:
Optional: It would be good to add a comment to reassure the reader that the only difference between these two implementations is `L'-'` vs. `'-'`.

Reply:
It seems that the _WIN32 version of `NEXTARG()` is not needed. We should not use `L'-'` on Windows.